src/inspector_agent.cc

```diff
@@ -658,9 +686,21 @@ bool Agent::StartIoThread(bool wait_for_connect) {
   }
 
   v8::Isolate* isolate = parent_env_->isolate();
+  HandleScope handle_scope(isolate);
+
+  // Enable tracking of async stack traces
+  if (!enable_async_hook_function_.IsEmpty()) {
+    Local<Function> enable_fn = enable_async_hook_function_.Get(isolate);
```

Comment:
Done in eb1bca, PTAL.

I have literally zero knowledge about `ConnectJSBindingsSession`. I think it would be good to add a test to cover this scenario, but I have no idea how such test should look like. Any ideas?